tableslurp

```diff
@@ -133,12 +133,22 @@ class DownloadHandler(object):
         json_data = json.loads(key.get_contents_as_string())
         self.fileset = json_data[self.origin]
         log.info('Fileset contains %d files to download' % (len(self.fileset)))
-        k = bucket.get_key('%s/%s' % (self.prefix, self.fileset[0]))
+        k = None
+        idx = 0
+        for idx in range(0, len(self.fileset)):
+			k = bucket.get_key('%s/%s' % (self.prefix, self.fileset[idx]))
+			if k != None:
+				break
```

Comment:
You are mixing tabs and spaces for indentation here and below. Tablesnap overall appears to use spaces for indentation. This will likely cause at least unintended behavior, if not errors.